### PR TITLE
node, ir: set default dial timeout to 1 minute

### LINKED ISSUE #2891: default timeouts are extremely aggressive
1. connection timeouts less than 10s are extremely aggressive and leads to connection breaks
2. connection timeouts should be the same across all services
3. i'd recommend to set all network timeouts to 1 minute

### PR DESCRIPTION
**Pull request by End-rey**
Closes #2891.

Question about these timeouts:
- in node:
   - `apiclient.stream_timeout`
   - `policer.head_timeout`
- in inner ring:
   - `audit.timeout.*`

Should I do the same for them?

### CODE REVIEW

**Review by roman-khimov — COMMENTED** [review]
Stream timeout should be OK to raise. Policer is a bit more tricky, some 10-15s is OK for it, but 1m seems to be too much. Audit better be left as is.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ attribute, which is used for container domain name in NNS contracts (#2954)
 - Metabase graveyard scheme (#2929)
 - When an error is returned, no additional help output is displayed in cobra-based programs (#2942)
 - Use org-wide linter (#2943)
+- Default dial timeout to one minute (#2963)
```
Comment (carpawell) [inline CHANGELOG.md:24]: can we also have it changeloged in `Updating from` section?

**Review by carpawell — COMMENTED** [review]
> Stream timeout should be OK to raise

@roman-khimov the only problem I know about such timeouts raising is bad failover performance: once a node is down too many routines are waiting for too long before they think the node is dead.

Head/audit timeouts are also ok to me to be kept

**Comment by roman-khimov** [comment]
> raising is bad failover performance

Sure. Can be adjusted for fast networks. Mainnet/testnet are likely to be OK with 1m.

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +41,11 @@ introduced in version 0.22.3 and support for binary keys was removed from
 other components in 0.33.0 and 0.37.0. Please migrate to wallets (see 0.37.0
 notes) if you've not done it previously.
 
+Increace default timeout for dialing connections in node: `morph.dial_timeout`,
```
Comment (carpawell) [inline CHANGELOG.md:44]:
```suggestion
Increase default timeout for dialing connections in node: `morph.dial_timeout`, 
```

**Review by carpawell — APPROVED** [review]
Otherwise ok

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**